packages/elements/src/containers/mediaQueries.ts

```diff
@@ -1,5 +1,6 @@
 export const mediaQueries = {
-  MOBILE: "@media only screen and (min-width: 320px) and (max-width: 1030px)",
+  MOBILE: "@media only screen and (max-width: 1030px)",
```

Comment:
Removed the min 320px here, I think there are very very few mobile devices that are narrower than 320px, but if there were, it should still get mobile styles, right? Not sure why this was in there.